src/controllers/accounts.js

```diff
@@ -179,9 +180,16 @@ function * subscribeTransfers () {
   if (this.websocket.readyState !== 1) return
 
   const close = model.subscribeTransfers(name, this.req.user, (notification) => {
-    this.websocket.send(JSON.stringify(notification))
+    this.websocket.send(JSON.stringify(notification), (error) => {
+      log.error('failed to send notification to ' + this.req.ip, error)
+    })
   })
 
+  // Send a message upon connection
+  this.websocket.send(JSON.stringify({
+    type: 'connect'
```

Comment:
Since there's not a spec that I know of I'm not sure how official it is. Stefan added it in https://github.com/interledgerjs/five-bells-ledger/pull/336/commits/e761ad604a9ef9e3a737333f165b1b4d2bbb5330, though.